etn-ccis/blui-doc-it
# Add release notes

## Changes proposed in this Pull Request:

- Add release notes for our R26 release
-
-

## To Test:

- After `yarn && yarn start`, check:
  - the home page, where the release summary is
  - the detailed release notes
  - ~Note that packages marked with a `❗️` is waiting for master PR / publishing~
